### ISSUE TITLE
[very edge case] Privacy Badger just "got reset" after computer (WIn7) crashed, then "Start new seesion" in about:sessionrestore - also running Adblock Plus, HTTPS Everwhere, genneral shit-laptop.

### ISSUE THREAD

**Issue by ScreapDK** [EFForg/privacybadgerfirefox#708]
Sorry, if this is too edge case/narrow a bug.

My whole computer just crashed

**Comment by cooperq** [comment]
I agree that this would be an excellent feature for us to support. I think that the proper way to do this is to keep relying on a local nonsycned data store but allow the user to check a setting in privacy badger which will then copy the data to the firefox synced data store. I don't think we should do any of this work though until #208 is fixed.

**Comment by paoletto** [comment]
I second this very much. For people who uses FF Sync, and given the way privacy badger works ATM (default green), sync'ing user settings would prevent several mental breakdowns =)

**Comment by sebastianst** [comment]
This feature would be great!
I use FF on several devices and would also love to see the different privacy badgers to _learn together_. I am not sure if the op feature request only refers to the custom rules to be synchronized or also the learned heuristics?

**Comment by kenneho** [comment]
+1

**Comment by andromedarabbit** [comment]
+1 also
